dashboard/config/routes.rb

```diff
@@ -179,6 +179,7 @@ def redirect_to_teacher_dashboard
 
   # one-off internal reports
   get '/admin/brook/csppd', to: 'reports#csp_pd_responses', as: 'csp_pd_responses'
+  get '/admin/roxanne/hoc_signups', to: 'admin_reports#hoc_signups', as: 'hoc_signups'
```

Comment:
+1 on this idea, I think Alice and Laura were interested in this as well so it's not just for Roxanne